test_toy_factory.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+import toy_factory
+
+
+class ToyFactoryTests(unittest.TestCase):
+
+    def testIsBoat(self):
+        toy = toy_factory.ToyFactory.make_toy('boat')
```

Comment:
Miert nem importalod be az objektumot, akkor szerintem nem kell a `toy_factory.` ele.